frontend/src/pages/DetailedCourse/DetailedCourse.tsx

```diff
@@ -111,48 +104,36 @@ const DetailedCourse: React.FC<IDetailedCourse> = ({
               type={INFO.detailedCourse}
             />
           </CourseInfoBox>
-          {isLoading && targetId === buttonId.start ? (
-            <StartButton id={buttonId.start} variant="mediumOutlined" disabled>
+
+          {isLoading ? (
+            <CustomButton variant="mediumOutlined" disabled>
               <ButtonLoader buttonSpinner={buttonSpinner} />
-            </StartButton>
+            </CustomButton>
+          ) : page === PAGES.coursesList ? (
+            <CustomButton
+              color="primary"
+              variant="mediumContained"
+              onClick={(event) => handleApplyCourse(event)}
+            >
+              Apply the course
```

Comment:
Shouldn't we move button text to constant and reuse it here?

Reply:
maybe, it was before me